Sources/MongoSwift/WriteConcern.swift

```diff
@@ -3,6 +3,28 @@ import Foundation
 
 /// A class to represent a MongoDB write concern.
 public struct WriteConcern: Codable {
+    /// Majority WriteConcern
+    /// - SeeAlso: https://docs.mongodb.com/manual/reference/write-concern/#writeconcern._dq_majority_dq_
+    // swiftlint:disable force_try
```

Comment:
I think the docstring won't get associated with the property if theres the comment in between. I think you can use // `swiftlint:disable:this` to put the disable on the same line as the definition, or `swiftlint:disable:previous` to put it on the line after.
also when we disable this rule we try to add a comment saying why it's ok.